Fix for main window not being maximized because of the "choose algo" window
shown at startup.
--- a/VDF.GUI/Views/MainWindow.xaml.cs
+++ b/VDF.GUI/Views/MainWindow.xaml.cs
@@ -85,10 +88,10 @@ namespace VDF.GUI.Views {
 					"ScannedFiles.db")))
 					return;
 
-			var dlg = new Views.ChooseAlgoView();
 			while (!this.IsVisible) {
 				await Task.Delay(200);
 			}
+			var dlg = new Views.ChooseAlgoView();
 			await dlg.ShowDialog(this);
 			if (dlg.FindControl<RadioButton>("Cb16x16")?.IsChecked == true)
 				VDF.Core.Utils.DatabaseUtils.Create16x16Database();
